app/models/locations/map.rb

```diff
@@ -0,0 +1,99 @@
+module Locations
+  class Map
+    attr_accessor :id,:loc
+
+    def initialize(id: nil, loc: nil)
+      @locator_url = "http://library.princeton.edu/locator/index.php?"
```

Comment:
Could the locator_url and and stackmap_url instance variables each be separate methods? ie:
```ruby
def locator_url
  'http://library.princeton.edu/locator/index.php?'
end
```